### ISSUE TITLE
Accessibility: Object Status: Clickable information is not conveyed, when clickable object statuses are accessed.

### ISSUE THREAD

**Issue by puru-hk** [SAP/fundamental-ngx#3448]
Description: Clickable information is not conveyed, when clickable object statuses are accessed.

Expected: Clickable information about UI should be conveyed to the user

Screenshot:

![image](https://user-images.githubusercontent.com/32538291/94521255-5feb7000-024b-11eb-984f-d2f7505ca974.png)

**Comment by Lokanathan-k** [comment]
fixed as part of the PR https://github.com/SAP/fundamental-ngx/pull/4176
